File: 2.0/python/calls/getCall.py
from callfire.client import CallfireClient

client = CallfireClient('api-login', 'api-password')
response = client.calls.getCall(
    id=379506003,
    # return only specific fields
    fields='fromNumber,toNumber,modified,finalCallResult'
).result()

# see sample JSON response for this API
# on 'curl' samples tab
print(response)

File: 2.0/php/calls/getCall.php
<?php

class ApiClientSample {

    public static function main() {
        $client = \CallFire\Api\DocumentedClient::createClient("login", "password");
        $request = $client->getCall();
        $request->getOperationConfig()->setPathParameters(array("id" => 379506003));
        $request->getOperationConfig()->setQueryParameters(array("fields" => "fromNumber,toNumber,modified,finalCallResult"));
        $result = $client->request($request);
        $json = json_decode($result->getBody());
    }
}

ApiClientSample::main();
